buildtest/cli/__init__.py

```diff
@@ -260,7 +260,9 @@ def get_parser():
     )
     parser.add_argument("-r", "--report", help="Specify path to test report file")
 
-    subparsers = parser.add_subparsers(title="COMMANDS", dest="subcommands", metavar="")
+    subparsers = parser.add_subparsers(title="COMMANDS", dest="subcommands", metavar=""
```

Comment:
you removed a close parenthesis `)` on this line . Simply add it back

Reply:
i will do that